src/components/PageTemplate/Footer.jsx

```diff
@@ -3,24 +3,22 @@ import {Flex, Link, Text} from "@chakra-ui/react";
 import {Icon} from "@chakra-ui/icons";
 import {FaEnvelope, FaGithub} from 'react-icons/fa';
 
-const Footer = () => {
-    return (
-        <Flex direction="column" m={16} align="center">
-            <Flex align="center">
-                <Text m={2}>Contact us: </Text>
-                <Link href="https://github.com/InvestoBull/stock-price-prediction"
-                      isExternal={true}
-                      p={0} bg="transparent" borderRadius="full">
-                    <Icon as={FaGithub} w={5} h={5} m={2}/>
-                </Link>
-                <Link href="mailto:team.investobull@gmail.com"
-                      p={0} bg="transparent" borderRadius="full">
-                    <Icon as={FaEnvelope} w={5} h={5} m={2}/>
-                </Link>
-            </Flex>
-            Copyright © InvestoBull 2021
+const Footer = () => (
```

Comment:
I think we should either add a divider (i.e., a horizontal line), or change the background colour of the footer area, so that it stands out a bit from the page it's on

Reply:
@paulsgh uhm, why is that necessary? I think a page with a consistent theme is better. we can discuss this on saturday or something

Reply:
I think it adds structure to the pages; it doesn't have to be very abrupt (so it wouldn't interfere with the theme). You can look at github's footer and their minimal divider as an example.

But yes sounds good, let's discuss over a call

Reply:
added a divider